apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx

```diff
@@ -148,10 +189,60 @@ const RemoteSvgTest: React.FunctionComponent = () => {
   );
 };
 
+const CustomSvgInputTest: React.FunctionComponent = () => {
+  const defaultxml = `
+  <svg width="256" height="256" viewBox="0 0 48 48">
+  <rect width="48" height="48" fill="red"/>
+  </svg>
+`;
+  const [inputText, setInputText] = useState<string>(defaultxml);
+  const [svgText, setSvgText] = useState<string>(defaultxml);
+  const [dataText, setDataText] = useState<string>('click draw to see parsed data');
+  const [showParse, setShowParsed] = useState<boolean>(false);
+  return (
+    <View>
+      <Text>
+        Paste an svg in here and click Draw to try it out. The way it uses rnsvg&apos;s parse function to attempt to validate svg may cause
+        asserts and crashes if the svg is invalid. Looking for another solution for this, but this is still useful in its current state.
+      </Text>
+      <TextInput
+        multiline={true}
+        accessibilityLabel="Custom svg text input"
+        value={inputText}
+        style={{ height: 256, width: 1024, borderColor: 'gray', borderWidth: 1, fontSize: 16 }}
```

Comment:
Could we make this style object a const object outside of the component since it's not changing? That will keep it from being recreated on each render